nzedb/http/Capabilities.php

```diff
@@ -69,30 +70,28 @@ public function output($data, $params, $xml = true, $type = '')
 	{
 		$this->type = $type;
 
+		$options = [
+			'Parameters' => $params,
+			'Data'       => $data,
+			'Server'     => $this->getForMenu(),
+			'Type'       => $type
+		];
+
+		$response = (new XML_Response($options))->returnXML();
+
```

Comment:
Couldn't this go inside the IF. No point spending the cycles creating this, only to overwrite it if JSON is wanted.